liveblog.php

```diff
@@ -252,7 +252,16 @@ public static function init() {
 			 * we can possibly introduce this to other post types later.
 			 */
 			$post_types = array( 'post' );
-			self::$supported_post_types = apply_filters( 'liveblog_modify_supported_post_types', $post_types );
+
+			/**
+			 * Filters post types supported by liveblog.
+			 *
+			 * @since 2.0
+			 *
+			 * @param array An array of supported post types.
```

Comment:
The type is already specified as an array, so the description can just be `Supported post types.`